database/seed.js

```diff
@@ -42,54 +42,86 @@ const seedTiers = () => {
     tierId = i;
     tierOrder = 1;
     itemId = Math.floor(Math.random() * 48) + 1;
-    tierData.push ({
-      tierId : tierId,
-      itemId : itemId,
+    tierData.push({
+      tierId: tierId,
+      itemId: itemId,
       itemOrder: tierOrder += 1
     });
     itemId = Math.floor(Math.random() * 48) + 49;
-    tierData.push ({
-      tierId : tierId,
-      itemId : itemId,
+    tierData.push({
+      tierId: tierId,
+      itemId: itemId,
       itemOrder: tierOrder += 1
     });
-    if (Math.random() < .3 ) {
+    if (Math.random() < .3) {
       itemId = Math.floor(Math.random() * 48) + 97;
-      tierData.push ({
-        tierId : tierId,
-        itemId : itemId,
+      tierData.push({
+        tierId: tierId,
+        itemId: itemId,
         itemOrder: tierOrder += 1
       });
     }
-    if (Math.random() < .3 ) {
+    if (Math.random() < .3) {
       itemId = Math.floor(Math.random() * 48) + 145;
-      tierData.push ({
-        tierId : tierId,
-        itemId : itemId,
+      tierData.push({
+        tierId: tierId,
+        itemId: itemId,
         itemOrder: tierOrder += 1
       });
     }
-    if (Math.random() < .3 ) {
+    if (Math.random() < .3) {
       itemId = Math.floor(Math.random() * 48) + 193;
-      tierData.push ({
-        tierId : tierId,
-        itemId : itemId,
+      tierData.push({
+        tierId: tierId,
+        itemId: itemId,
         itemOrder: tierOrder += 1
       });
     }
-    if (Math.random() < .3 ) {
+    if (Math.random() < .3) {
       itemId = Math.floor(Math.random() * 48) + 241;
-      tierData.push ({
-        tierId : tierId,
-        itemId : itemId,
+      tierData.push({
+        tierId: tierId,
+        itemId: itemId,
         itemOrder: tierOrder += 1
-      })
+      });
     }
   }
   return tierData;
 }
 
+const seed = () => {
+  db.Bundles.drop()
+  .then(() => {
+    db.Bundles.sync({ force: true })
+    .then(() => {
+      let data = seedBundles();
+      db.Bundles.bulkCreate(data, { validate: true })
+      .then(() => {
+        console.log('bundles created');
+      })
+      .catch(err => {
+        console.log('failed to create bundles', err);
+      })
+    })
+  });
+
+  db.Tiers.drop()
+  .then(() => {
+    db.Tiers.sync({ force: true })
+    .then(() => {
+      let data = seedTiers();
+      db.Tiers.bulkCreate(data, { validate: true})
+      .then(() => {
+        console.log('tiers created');
+      })
+      .catch(err => {
+        console.log('failed to create tiers', err);
+      })
+    })
+  });
+};
 
+seed();
```

Comment:
This reorganization (i.e., taking this out of index and putting it in seed) makes a ton of sense to me.

Reply:
Yeah, I'd been planning to move it over once I did my refactor, but hadn't gotten to it until now.